pkg/question/question_test.go

```diff
@@ -1139,3 +1185,400 @@ func cleanupQuestionTest() string {
+		Vpcs: []*ec2.Vpc{
+			{
+				VpcId:     aws.String("vpc-12345"),
+				CidrBlock: aws.String("some block"),
+				Tags: []*ec2.Tag{
+					{
+						Key:   aws.String("Name"),
+						Value: aws.String("test vpc"),
+					},
+				},
+				IsDefault: aws.Bool(true),
+			},
+			{
+				VpcId:     aws.String("vpc-67890"),
+				CidrBlock: aws.String("some block"),
+				IsDefault: aws.Bool(false),
+			},
+			{
+				VpcId:     aws.String(defaultVpc),
+				CidrBlock: aws.String("some block"),
+				IsDefault: aws.Bool(false),
+			},
+			{
+				VpcId:     aws.String("vpc-41239"),
+				CidrBlock: aws.String("some block"),
+				IsDefault: aws.Bool(false),
+			},
+		},
+	}
+
+	answer, err := question.AskVpc(testEC2, defaultVpc)
+	th.Ok(t, err)
+	th.Equals(t, defaultVpc, *answer)
+
+	cleanupQuestionTest()
+}
+
+func TestAskSubnet_WithDefault(t *testing.T) {
+	const testVpc = "vpc-12345"
+	const defaultSubnet = "subnet-12345"
+	initQuestionTest(t, "\n")
+
+	testEC2.Svc = &th.MockedEC2Svc{
+		Subnets: []*ec2.Subnet{
+			{
+				SubnetId:         aws.String("subnet-01894"),
+				VpcId:            aws.String(testVpc),
+				CidrBlock:        aws.String("some block"),
+				AvailabilityZone: aws.String("some az"),
+				Tags: []*ec2.Tag{
+					{
+						Key:   aws.String("Name"),
+						Value: aws.String("test subnet"),
+					},
+				},
+			},
+			{
+				SubnetId:         aws.String("subnet-67890"),
+				VpcId:            aws.String(testVpc),
+				CidrBlock:        aws.String("some block"),
+				AvailabilityZone: aws.String("some az"),
+			},
+			{
+				SubnetId:         aws.String("subnet-77245"),
+				VpcId:            aws.String(testVpc),
+				CidrBlock:        aws.String("some block"),
+				AvailabilityZone: aws.String("some other az"),
+			},
+			{
+				SubnetId:         aws.String(defaultSubnet),
+				VpcId:            aws.String(testVpc),
+				CidrBlock:        aws.String("some block"),
+				AvailabilityZone: aws.String("some az"),
+			},
+		},
+	}
+
+	answer, err := question.AskSubnet(testEC2, testVpc, defaultSubnet)
+	th.Ok(t, err)
+	th.Equals(t, defaultSubnet, *answer)
+
+	cleanupQuestionTest()
+}
+
+func TestAskSubnetPlaceholder_WithDefault(t *testing.T) {
+	const defaultAz = "us-east-2"
+	initQuestionTest(t, "\n")
+
+	testEC2.Svc = &th.MockedEC2Svc{
+		AvailabilityZones: []*ec2.AvailabilityZone{
+			{
+				ZoneName: aws.String("us-east-1"),
+				ZoneId:   aws.String("some id"),
+			},
+			{
+				ZoneName: aws.String(defaultAz),
+				ZoneId:   aws.String("some id"),
+			},
+			{
+				ZoneName: aws.String("us-west-1"),
+				ZoneId:   aws.String("some id"),
+			},
+			{
+				ZoneName: aws.String("us-west-2"),
+				ZoneId:   aws.String("some id"),
+			},
+		},
+	}
+
+	answer, err := question.AskSubnetPlaceholder(testEC2, defaultAz)
+	th.Ok(t, err)
+	th.Equals(t, defaultAz, *answer)
+
+	cleanupQuestionTest()
+}
+
+func TestAskBootScriptConfirmation_WithDefault(t *testing.T) {
+	defaultBootScript := ""
+	defaultConfirmation := cli.ResponseNo
```

Comment:
`ResponseNo` [is the initial default anyway](https://github.com/GavinBurris42/aws-simple-ec2-cli/blob/main/pkg/question/question.go#L1230), so this test won't prove that the override mechanism works. Can we rewrite to use `ResponseYes`?

```suggestion
	defaultConfirmation := cli.ResponseYes
```